### DISCUSSION TITLE
Best practice feature request: touch and wheel event listeners as `passive`

### DISCUSSION THREAD

**Discussion by PolGuixe** [google/model-viewer#2206] [Q&A]
### Description
Would it be possible to have a prop to enable/disable the touch and wheel event listeners as `passive`?

It is recommended by Lighthouse
![Screenshot 2020-05-13 at 17 45 01](https://user-images.githubusercontent.com/6626225/81835163-3226b880-9542-11ea-8855-757322290918.png)

More info [here](https://web.dev/uses-passive-event-listeners/?utm_source=lighthouse&utm_medium=lr)

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [ ] Windows

### Versions

- model-viewer: v0.10.0
- three.js: r115

**Comment by PolGuixe** [comment]
The warning disapears  in Lighthouse if you are using the NPM package, instead of the Unpkg ones.

**Comment by elalish** [comment]
That is quite strange. We can't mark these as passive because they call preventDefault(); it'd be strange if your page scrolled when you moved the model. Lighthouse says it filters out these listeners, but it's clearly failing. I'll file a bug on them.
